cmake/IgnSetCompilerFlags.cmake

```diff
@@ -115,11 +115,18 @@ endmacro()
 # Set up compilation flags for GCC or Clang
 macro(ign_setup_gcc_or_clang)
 
+  option(USE_DEFAULT_VISIBILITY_HIDDEN "Hide symbols by default if they are not explicitly specified as visible" FALSE)
```

Comment:
I'm getting this error:
```

CMake Warning (dev) at /home/addisu/ws/dome/install/share/cmake/ignition-cmake2/cmake2/IgnSetCompilerFlags.cmake:118 (option):
  Policy CMP0077 is not set: option() honors normal variables.  Run "cmake
  --help-policy CMP0077" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  For compatibility with older versions of CMake, option is clearing the
  normal variable 'USE_DEFAULT_VISIBILITY_HIDDEN'.
Call Stack (most recent call first):
  /home/addisu/ws/dome/install/share/cmake/ignition-cmake2/cmake2/IgnSetCompilerFlags.cmake:59 (ign_setup_gcc_or_clang)
  /home/addisu/ws/dome/install/share/cmake/ignition-cmake2/cmake2/IgnConfigureBuild.cmake:86 (ign_set_compiler_flags)
  CMakeLists.txt:92 (ign_configure_build)
This warning is for project developers.  Use -Wno-dev to suppress it.

```

Reply:
what version of cmake are you using? and what OS?

Reply:
Ah, I should have mentioned that. I'm on Ubuntu 20.04, cmake 3.16.3

Reply:
I changed it to an argument to `ign_configure_build` instead of an `option` in https://github.com/ignitionrobotics/ign-cmake/pull/196/commits/fd38458199dd12efb2239c7853159a0096f6e55b. I think that resolves the warning

Reply:
Yup, that resolved it for me. Thanks!